ShipperOptions.json

```diff
@@ -1985,6 +1985,20 @@
         "dim_count": 2,
         "max_oz": 13
       },
+      {
+        "value": "LGENV",
+        "display": "Large Envelope",
+        "max_dims": [15, 12, 0.75],
+        "dim_count": 2,
+        "max_oz": 15
+      },
+      {
+        "value": "NMLETTER",
```

Comment:
in shipper code, if this is a FCMI label, we will translate this from `NMLETTER` to `NMLTR` is that going to be an issue when serializing things out?

Reply:
It's the same for `LETTER`, for FCMI service type they shortened to LTR for some reason.

Reply:
maybe. We could add the short versions here as well so we have something to map value to display.

Reply:
I didn't want to have multiples Parcels in the dropdown. would that get filtered out? The LGENV and FLAT are especially weird for service types.

Reply:
oh, i guess we're translating in the shipper code to send, but we aren't actually changing the values that we send. maybe we don't have to worry about serialization.

Reply:
oh, I thought it was shipping method. Yeah with package we can't have duplicates/alts.